src/components/molecules/NavBar/playa.scss

```diff
@@ -37,7 +37,6 @@ $sand: #937c63;
     height: 100%;
     margin-left: auto;
     margin-right: auto;
-    background-color: transparent;
 
     .navbar-logo_container {
       display: flex;
```

Comment:
Potentially/probably OOS for this issue specifically, but we should really be cleaning up/removing all of these extra styles from this `playa.scss` file. Eg. `.navbar-container` should entirely be defined within `src/components/molecules/NavBar/NavBar.scss`, so it shouldn't really exist in `src/components/molecules/NavBar/playa.scss`, etc.

Reply:
yeah, agree, there is a `@debt` comment in the file about that.
https://github.com/sparkletown/sparkle/blob/2ab1f73ef4f7eab79f07c2788a32525761310736/src/components/molecules/NavBar/playa.scss#L18

Reply:
Sounds good